MiniApp/Classes/core/Extensions/NotificationCenter+RAT.swift

```diff
@@ -3,21 +3,23 @@ internal extension NotificationCenter {
         sendAnalytics(event: event, type: type, parameters: customData)
```

Comment:
this method will never send anything from now because it won't enter in the `for in` loop line 10

Reply:
Removed them as disucussed